src/versions.jl

```diff
@@ -16,6 +16,6 @@
 #   is *not* accepted. Presumably CUDA runtime follows semantic
 #   versioning so this can automatically be set to the next major
 #   version.
-const onnxruntime_version = v"1.20.1"
+const onnxruntime_version = v"1.20.2"
```

Comment:
```suggestion
const onnxruntime_version = v"1.20.1"
```
Sorry I was talking about the ONNXRunTime.jl package version, not the binary version.